src/frontend/planner_test/tests/testdata/output/recursive_cte.yaml

```diff
@@ -1,14 +1,24 @@
 # This file is automatically generated. See `src/frontend/planner_test/README.md` for more information.
 - name: basic
   sql: WITH RECURSIVE t1 AS (SELECT 1 AS a UNION ALL SELECT a + 1 FROM t1 WHERE a < 10) SELECT * FROM t1;
-  planner_error: |-
-    Feature is not yet implemented: recursive CTE is not supported
-    Tracking issue: https://github.com/risingwavelabs/risingwave/issues/15135
+  logical_plan: |-
+    LogicalProject { exprs: [$expr1] }
+    └─LogicalRecursiveUnion { id: 4 }
+      ├─LogicalProject { exprs: [1:Int32] }
+      │ └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
+      └─LogicalProject { exprs: [(1:Int32 + 1:Int32) as $expr1] }
```

Comment:
Why it's `1+1` instead of `a+1`?

Reply:
I raised the same question just above. 😄

ps. maybe due to the on-the-fly constant folding, but I'm not sure.